fontforge/python.c

```diff
@@ -810,6 +810,114 @@ static PyObject *PyFF_UnicodeNamesListVersion(PyObject *UNUSED(self), PyObject *
     return( ret );
 }
 
+/* Ligature & Fraction information based on current Unicode (builtin) chart. */
+/* Unicode chart seems to distinguish vulgar fractions from other fractions. */
+/* These routines test value with internal table. Returns true/false values. */
```

Comment:
Everything is all in the same Pull request.
There are the bare-bone functions in unicode library. Then there are working script functions in native scripting and also in python scripting. Then there are actual test scripts in the testsuite as working examples. And then there is documentation in scripting and python html help files.